crates/uv-cli/src/lib.rs

```diff
@@ -567,6 +567,19 @@ pub enum ProjectCommand {
     )]
     Remove(RemoveArgs),
     /// Update the project's environment (experimental).
+    ///
+    /// Syncing ensures that all dependencies of the project are installed and
```

Comment:
"project dependencies" instead of "dependencies of the project"

Reply:
```suggestion
    /// Syncing ensures that all project dependencies are installed and
```